### DISCUSSION TITLE
brew install nugget-doom

### DISCUSSION THREAD

**Discussion by noahastor** [Homebrew/discussions#5362] [Everyday usage] [answered]
### Output of `brew config`

```shell
/
```

### Output of `brew doctor`

```shell
/
```

### Description of issue

brew install nugget-doom
==> Downloading https://formulae.brew.sh/api/formula.jws.json
==> Downloading https://formulae.brew.sh/api/cask.jws.json
Warning: No available formula with the name "nugget-doom"

I was wondering if a formula could be created for this woof-doom fork.

Thank you.

**Comment by SMillerDev** [comment, marked as answer]
> I was wondering if a formula could be created for this woof-doom fork.

[Yes it can](https://docs.brew.sh/Adding-Software-to-Homebrew)

**Reply by noahastor** [reply]
Thank you.
